src/pages/Detail/CommonDetail/ProductContentBottom.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import * as P from './ProductDetailStyle';
+import { BiPlusMedical } from 'react-icons/bi';
+import { ImMinus } from 'react-icons/im';
+
+export default function ProductContentBottom({ detailContent }) {
+  const [count, setCount] = useState(1);
+
+  const handleQuantity = type => {
+    if (type === 'plus') {
+      setCount(count + 1);
+    } else if (type === 'minus') {
+      if (count === 1) {
+        return;
+      }
+      setCount(count - 1);
+    }
+  };
+
+  return (
+    <P.ProductContentBottom>
+      <P.ProductQuantity>
+        <P.QuantityContent>총 수량</P.QuantityContent>
+        <P.QuantityCountBox>
+          <ImMinus className="icon" onClick={() => handleQuantity('minus')} />
+          <P.QuantityCount>{count}</P.QuantityCount>
+          <BiPlusMedical
+            className="icon"
+            onClick={() => handleQuantity('plus')}
+          />
+        </P.QuantityCountBox>
+      </P.ProductQuantity>
+      <P.ProductTotalPriceBox>
+        <P.ProductTotalPriceText>주문금액</P.ProductTotalPriceText>
+        <P.ProductTotalPrice>
+          <span className="bold">{detailContent.price * count}</span>원
```

Comment:
```suggestion
          <span className="bold">
            {(detailContent.price * count).toLocaleString()}
          </span>
          원
```
참고로 UI적으로 조금 더 좋게 보여주는 부분으로 이와 같은 메서드를 적용할 수 있겠습니다.